imports/api/tasks.js

```diff
@@ -0,0 +1,72 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { check } from 'meteor/check';
+
+export const Tasks = new Mongo.Collection('tasks');
+
+if (Meteor.isServer) {
+  // This code only runs on the server
+  // Only publish tasks that are public or belong to the current user
+  Meteor.publish('tasks', function tasksPublication() {
+    return Tasks.find({
+      $or: [
+        { private: { $ne: true } },
```

Comment:
We don't have anything with private tasks right?
Also, we haven't implemented users yet, so there wouldn't be any user ids correct?